build_library/toolchain_util.sh

```diff
@@ -52,6 +56,7 @@ get_portage_arch() {
         m68*)       echo m68k;;
         mips*)      echo mips;;
         powerpc64*) echo ppc64;;
+        ppc64le*) echo ppc64le;;
```

Comment:
For official gentoo portage, ppc64le and ppc64 are using the same keywords "ppc64".
So above seems wrong and should be changed to "powerpc64le-*) echo ppc64le".